app/assets/javascripts/components/locator-map.js

```diff
@@ -27,7 +27,26 @@
 
   var LocatorMap = function(options) {
     this.$container = options.$container;
-    this.$offices = options.$offices;
+    this.$locations = options.$locations;
+
+    var iconDefaults = {
+      iconAnchor: [17, 48],
+      popupAnchor: [0, -48],
+      shadowUrl: this.$container.data('markerShadowPath'),
+      shadowAnchor: [13, 41]
+    };
+
+    this.icons = {
+      'transportation-office': L.icon($.extend({}, iconDefaults, {
+        iconUrl: this.$container.data('markerTransportationOfficeIconPath'),
+        iconRetinaUrl: this.$container.data('markerTransportationOfficeIcon2xPath')
+      })),
+
+      'weight-scale': L.icon($.extend({}, iconDefaults, {
+        iconUrl: this.$container.data('markerWeightScaleIconPath'),
+        iconRetinaUrl: this.$container.data('markerWeightScaleIcon2xPath')
+      }))
+    };
```

Comment:
Now that we've got multiple icons on the map, abstract common options into a defaults object and merge that with the specific options for each. Save all that as a `this.icons` object and reference it later on when creating markers.